packages/material-ui/src/Select/Select.test.js

```diff
@@ -671,6 +671,36 @@ describe('<Select />', () => {
 
       expect(getByRole('listbox')).to.be.ok;
     });
+
+    it('open only with the left mouse button click', () => {
+      // Test for https://github.com/mui-org/material-ui/issues/19250#issuecomment-578620934
+      // Right/middle mouse click shouldn't open the select
+      const { getByRole, queryByRole, getByTestId } = render(
+        <Select
+          MenuProps={{
```

Comment:
Now we can get rid of these and then we have a neat little test :+1: 

Sorry for these chores but tests with we had some bad experience with tests that included unrelated things. Always requires extra attention when refactoring so we try to get it right the first time.

Reply:
OK, no problem!
Thank you for guiding thru UT polishing process!